stablehlo/dialect/TypeInference.cpp

```diff
@@ -1055,6 +1055,12 @@ LogicalResult inferOptimizationBarrierOp(
   return success();
 }
 
+LogicalResult inferOutfeedOp(Optional<Location> location, Value token,
+                             SmallVectorImpl<Type>& inferredReturnTypes) {
+  inferredReturnTypes.push_back(token.getType());
```

Comment:
Let's drop `token` as well and just return `TokenType::get` to make things a bit more robust. Sorry - didn't notice this during the first round of reviews.

Reply:
Sure, but I'd instead need to pass `MLIRContext`. Though, using this will make this similar to the other PRs recently merged.